src/lab.rs

```diff
@@ -401,22 +401,22 @@ mod test {
     #[test]
     fn red() {
         let a = Lab::from(Rgb::new(1.0, 0.0, 0.0));
-        let b = Lab::new(53.23288 / 100.0, 80.09246 / 128.0, 67.2031 / 128.0);
-        assert_relative_eq!(a, b, epsilon = 0.0001);
+        let b = Lab::new(53.23288, 80.09246, 67.2031);
+        assert_relative_eq!(a, b, epsilon = 0.01);
```

Comment:
I don't know the source of this data and this does not pass with 0.001. Anyway the Lab - Rgb should be taken care of in the color checker data, which is more accurate.